### PR TITLE
Experince Card and Experience page ui added

### PR DESCRIPTION
**Pull request by yashghuge007**

### CODE REVIEW

File: src/App/index.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { Routes, Route } from 'react-router-dom'
 import { Header, Sidebar } from '../components'
 import { Dashboard, Profile } from '../pages'
+import IntExp from '../pages/InterviewExperience'
```
Comment (jatinkh25) [inline src/App/index.tsx:4]: add InterviewExperience page in index.ts file of pages folder and remover this import. Import should be like dashboard and Profile page.

File: src/components/Cards/BasicCard/BasicCard.module.scss [hunk 2]
```diff
@@ -0,0 +1,117 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap');
```
Comment (jatinkh25) [inline src/components/Cards/BasicCard/BasicCard.module.scss:1]: remove this font family import its already done in dev in vite.config.ts

File: src/components/Cards/BasicCard/BasicCard.module.scss [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+        .lower_right{
+            width: auto;
+            text-align: right;
+            margin-top: 10px;
+            font-family: 'poppins';
+        }
+    }
+}
+
```
Comment (jatinkh25) [inline src/components/Cards/BasicCard/BasicCard.module.scss:95]: remove extra spaces. A space of one line is enough.

File: src/components/Cards/BasicCard/BasicCard.module.scss [hunk 4]
```diff
@@ -0,0 +1,117 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap');
+
```
Comment (jatinkh25) [inline src/components/Cards/BasicCard/BasicCard.module.scss:2]: remove extra spaces. A space of one line is enough.

File: src/components/Cards/BasicCard/BasicCard.module.scss [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+
+
+.truncate1 {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: clip;
+}
+
```
Comment (jatinkh25) [inline src/components/Cards/BasicCard/BasicCard.module.scss:109]: here also

File: src/components/PastExperience/index.tsx [hunk 6]
```diff
@@ -1,33 +1,31 @@
 import { Link } from 'react-router-dom'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faUser } from '@fortawesome/free-solid-svg-icons'
+import { Fragment } from 'react'
```
Comment (jatinkh25) [inline src/components/PastExperience/index.tsx:4]: remove this Fragment import if not using it.

File: src/components/UpcomingDrive/index.tsx [hunk 7]
```diff
@@ -1,33 +1,31 @@
 import { Link } from 'react-router-dom'
 import { faApple } from '@fortawesome/free-brands-svg-icons'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { Fragment } from 'react'
```
Comment (jatinkh25) [inline src/components/UpcomingDrive/index.tsx:4]: here also

File: src/pages/InterviewExperience/InterviewExperience.module.scss [hunk 8]
```diff
@@ -0,0 +1,69 @@
+.display {
+    display: flex;
+    font-family: 'poppins';
```
Comment (jatinkh25) [inline src/pages/InterviewExperience/InterviewExperience.module.scss:3]: remove this font family. Its already applied globally.

File: src/pages/InterviewExperience/InterviewExperience.module.scss [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+    .middle_right {
+        width: 20%;
+        background-color: beige;
+        
+
+        .mrbody {
+        margin: 15px 15px 15px 15px;
+
+            .Filters{
+
```
Comment (jatinkh25) [inline src/pages/InterviewExperience/InterviewExperience.module.scss:36]: remove extra spaces

File: src/pages/InterviewExperience/index.tsx [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+      <div className={styles.display}>
+        <div className={styles.middle_left}>
+          <div className={styles.head}>
+            <h3>Interview Experiences</h3>
+          </div>
+          <div className={styles.mlbody}>
+            {userIntExp.map((user: BasicCardProps) => (
+              <div key={user.id} className={styles.card_margins}>
+                <BasicCard
+                  id={user.id}
```
Comment (jatinkh25) [inline src/pages/InterviewExperience/index.tsx:18]: don't pass these  attributes manually. You can use spread operator for passing all the props directly like {...user}

**Review by jatinkh25 — CHANGES_REQUESTED** [review]

File: src/App/index.tsx (hunk 1 above)
Comment (yashghuge007) [reply src/App/index.tsx:4]: done

File: src/components/Cards/BasicCard/BasicCard.module.scss (hunk 2 above)
Comment (yashghuge007) [reply src/components/Cards/BasicCard/BasicCard.module.scss:1]: done

File: src/components/PastExperience/index.tsx (hunk 6 above)
Comment (yashghuge007) [reply src/components/PastExperience/index.tsx:4]: compnent of shawak, i have deleted this section

**Review by jatinkh25 — APPROVED** [review]

**Pull request merged**